src/deep-kernel/lib/Config/Driver/AsyncConfig.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Created by AlexanderC on 3/7/16.
+ */
+
+'use strict';
+
+import {AbstractDriver} from './AbstractDriver';
+import {HttpDriver} from './HttpDriver';
+import {FsDriver} from './FsDriver';
+
+export class AsyncConfig extends AbstractDriver {
+  /**
+   * @param {Kernel|*} kernel
+   * @param {String} configFile
+   */
+  constructor(kernel, configFile = AsyncConfig.DEFAULT_CONFIG_FILE) {
+    super();
+
+    this._kernel = kernel;
+    this._configFile = configFile;
+  }
+
+  /**
+   * @returns {Kernel|*}
+   */
+  get kernel() {
+    return this._kernel;
+  }
+
+  /**
+   * @returns {String|*}
+   */
+  get configFile() {
+    return this._configFile;
+  }
+
+  /**
+   * @param {String} configFile
+   * @private
+   */
+  _load(configFile = null) {
+    this._configFile = configFile || this._configFile;
+
+    this._kernel.isBackend ? this._loadFromFS() : this._loadFromEndpoint();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/MitocGroup/deep-framework/file/2210381243/issues/source?bid=2895475&fileBranchId=3171000#l44)